src/renderer/base/renderer.cpp

```diff
@@ -1340,6 +1342,11 @@ void Renderer::_ScrollPreviousSelection(const til::point delta)
         {
             rc += delta;
         }
+
+        if (_currentCursorOptions)
+        {
+            _currentCursorOptions->coordCursor += delta;

```

Comment:
wat? why weren't we doing this before

Reply:
`_currentCursorOptions` is only a member to smuggle state up outside a function call and back into an entirely unrelated one at a later time. It's never used beyond that. I.e. it wasn't preserved between render passes.